modules/olm-catalogsource.adoc

```diff
@@ -26,10 +26,15 @@ metadata:
  namespace: olm
 spec:
  sourceType: grpc
- image: quay.io/operator-framework/upstream-community-operators:latest
+ image: quay.io/operator-framework/upstream-community-operators:latest <1>
```

Comment:
The community operator image should be `registry.redhat.io/redhat/community-operator-index:latest`

Reply:
I was trying to show the CatalogSource for OperatorHub.io, but it looks like it might actually should be `quay.io/operatorhubio/catalog:latest`.

From https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml found from https://operatorhub.io/how-to-install-an-operator#How-do-I-install-an-Operator-from-OperatorHub.io? (unless this is out of date):

```
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: operatorhubio-catalog
  namespace: olm
spec:
  sourceType: grpc
  image: quay.io/operatorhubio/catalog:latest
  displayName: Community Operators
  publisher: OperatorHub.io
```

Reply:
Ok, yes, this image `quay.io/operatorhubio/catalog:latest` is for the OperatorHub.io. And, this image `registry.redhat.io/redhat/community-operator-index:latest` is for the operator from the Community in OCP 4.6.